ENIGMAsystem/SHELL/Platforms/xlib/XLIBmain.cpp

```diff
@@ -22,6 +22,7 @@
 #include "XLIBmain.h"
 #include "LINUXjoystick.h"
 #include "XLIBwindow.h"
+#include "XDisplayGetters.h"
```

Comment:
This include doesn't technically do anything.

Reply:
It does now

Reply:
Nothing in XLIBmain uses display_get_width/display_get_height, so there is no need to include it. Having a source include functions it doesn't use does not change the emitted byte code. In the future, this will be a build warning.

Reply:
you should refresh the page more often